### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by dnadales** [rubik/stack-hpc-coveralls#19]
Compilation fails with the following error:

```
--  While building package stack-hpc-coveralls-0.0.4.0 using:
      C:\sr\setup-exe-cache\x86_64-windows\Cabal-simple_Z6RU0evB_1.24.2.0_ghc-8.0.2.exe --builddir=.stack-work\dist\ca59d0ab build --ghc-options " -ddump-hi -ddump-to-file"
    Process exited with code: ExitFailure 1
    Logs have been written to: C:\Users\NADALE~1\AppData\Local\Temp\hpc-test\.stack-work\logs\stack-hpc-coveralls-0.0.4.0.log

    Configuring stack-hpc-coveralls-0.0.4.0...
    Building stack-hpc-coveralls-0.0.4.0...
    Preprocessing library stack-hpc-coveralls-0.0.4.0...
    [1 of 6] Compiling SHC.Types        ( src\SHC\Types.hs, .stack-work\dist\ca59d0ab\build\SHC\Types.o )
    [2 of 6] Compiling SHC.Utils        ( src\SHC\Utils.hs, .stack-work\dist\ca59d0ab\build\SHC\Utils.o )
    [3 of 6] Compiling SHC.Stack        ( src\SHC\Stack.hs, .stack-work\dist\ca59d0ab\build\SHC\Stack.o )
    [4 of 6] Compiling SHC.Lix          ( src\SHC\Lix.hs, .stack-work\dist\ca59d0ab\build\SHC\Lix.o )
    [5 of 6] Compiling SHC.Coverage     ( src\SHC\Coverage.hs, .stack-work\dist\ca59d0ab\build\SHC\Coverage.o )
    [6 of 6] Compiling SHC.Api          ( src\SHC\Api.hs, .stack-work\dist\ca59d0ab\build\SHC\Api.o )

    C:\Users\NADALE~1\AppData\Local\Temp\stack2212\stack-hpc-coveralls-0.0.4.0\src\SHC\Api.hs:48:36: error:
        Variable not in scope:
          checkStatus
            :: ASetter Options b a0 (Maybe (t2 -> t1 -> t0 -> Maybe a1))

```

**Comment by rubik** [comment]
That's because `checkStatus` has been renamed to `checkResponse` in the newer version of Wreq ([ref](https://github.com/bos/wreq/blob/7dd23318b1ae1bb8d16431ce286b7d2245c8d960/changelog.md)). I'll fix it and upload a new version to Hackage.

**Comment by dnadales** [comment]
Ok, let me know if you need any help with that (I'll be available again on August 15). Thanks!

**Comment by bensteinberg** [comment]
Hi -- I just ran into this, too. +1 on uploading the new version. Thanks!

**Comment by rubik** [comment]
Fixed by #21
